xarray/ufuncs.py

```diff
@@ -78,10 +79,49 @@ def __call__(self, *args, **kwargs):
         return res
 
 
+def _skip_signature(doc):
+    if not isinstance(doc, str):
+        return doc
+
+    if doc.startswith(name):
```

Comment:
Where's `name` from? (Am I missing something very obvious?)

Reply:
no, you're not, that's a bug